starters/vue3-xstate-css/src/App.vue

```diff
@@ -0,0 +1,3 @@
+<template>
+	<RouterView />
```

Comment:
I would actually wrap this in MAIN, so that all the view loaded will be within main sop:

```
<template>
  <main>
    <RouterView />
  </main>
</tempalte>
```